portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -15,14 +15,14 @@
      *
      */
     private static final long serialVersionUID = 1L;
-    private int views = 0;
+    //private int views = 0;
```

Comment:
We want to try not to leave bits and pieces of code hanging when possible, so if you aren't planning on using this, I would suggest deleting it altogether (same for commented code below)